packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -25,25 +26,45 @@ declare global {
 /**
  * The table column for displaying string fields as text.
  */
-export class TableColumnText extends mixinTextBase(
-    TableColumnTextBase<TableColumnTextColumnConfig>
+export class TableColumnText extends mixinCustomSortOrderColumnAPI(
+    mixinTextBase(
+        TableColumnTextBase<
+        TableColumnTextColumnConfig,
+        TableColumnTextValidator
+        >
+    )
 ) {
     public placeholderChanged(): void {
         this.columnInternals.columnConfig = {
             placeholder: this.placeholder
         };
     }
 
-    protected override getColumnInternalsOptions(): ColumnInternalsOptions {
+    /** @internal */
+    public override getDefaultSortFieldName(): string | undefined {
+        return this.fieldName;
+    }
+
+    /** @internal */
+    public override getDefaultSortOperation(): TableColumnSortOperation {
+        return TableColumnSortOperation.localeAwareCaseSensitive;
+    }
+
+    protected override getColumnInternalsOptions(): ColumnInternalsOptions<TableColumnTextValidator> {
         return {
             cellRecordFieldNames: ['value'],
             cellViewTag: tableColumnTextCellViewTag,
             groupHeaderViewTag: tableColumnTextGroupHeaderViewTag,
             delegatedEvents: [],
             sortOperation: TableColumnSortOperation.localeAwareCaseSensitive,
```

Comment:
Actually, making them property getters seems to break things with the mixin. I'm going to change them back to functions, at least for now.

In case anyone is interested, the problem I'm running into is this:
When the base class for a column is generated with the mixins, the type for it ends up containing the following lines:
```ts
readonly defaultSortFieldName: string | undefined;
readonly defaultSortOperation: TableColumnSortOperation;
```
even though those properties are defined in the mixin as:
```ts
public abstract get defaultSortFieldName(): string | undefined;
public abstract get defaultSortOperation(): TableColumnSortOperation;
```
As a result, trying to override them in the concrete class as property getters causes a TypeScript compile error due to [this TypeScript rule](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#properties-overriding-accessors-and-vice-versa-is-an-error).

Reply:
I tried adding similar abstract properties to the `SortableColumn` mixin class locally, and I'm not seeing the errors you are. I'm not sure what I'm doing differently.

Reply:
I think I didn't see the problems when I built `nimble-components`. It wasn't until the full build ran and things tried to build on top of `nimble-components` such as `spright` and `nimble-angular`.